decidim-comments/lib/decidim/comments/query_extensions.rb

```diff
@@ -10,18 +10,16 @@ module QueryExtensions
       # type - A GraphQL::BaseType to extend.
       #
       # Returns nothing.
-      def self.extend!(type)
-        type.define do
-          field :commentable do
-            type !CommentableType
+      def self.define(type)
+        type.field :commentable do
+          type !CommentableType
 
-            argument :id, !types.String, "The commentable's ID"
-            argument :type, !types.String, "The commentable's class name. i.e. `Decidim::ParticipatoryProcess`"
+          argument :id, !types.String, "The commentable's ID"
+          argument :type, !types.String, "The commentable's class name. i.e. `Decidim::ParticipatoryProcess`"
```

Comment:
Is `Decidim::ParticipatoryProcess` actually a commentable? 😜

Reply:
No! But again, out of the scope of the PR